### DISCUSSION TITLE
Can I use this patches for YTM?

### DISCUSSION THREAD

**Discussion by MBYo** [kitadai31/revanced-patches-android6-7#32] [General]
Hello, I have Android 7.0 for now I succeeded with YT v.17.34.36 using your patches, but I Failed with YTM v.5.39.52.
I already installed YTM v.5.39.52 but won't show up in Revanced Manager so i couldn't patch it. 
Can you help me @kitadai31?

Thanks in advance

**Comment by kitadai31** [comment]
After changing the source to kitadai31, did you leave it as is?
You have to reset source to patch YTM.

---

Tips: Actually, you can use latest YTM apk (e.g. 6.02.54) for patching.
Through turning on Settings > Experimental patch support toggle.
5.39.52 is a little old.

**Comment by kitadai31** [comment]
Sorry, I misreaded your question.

This patches does not include YTM patches.
Because original YouTube Music still supports Android 5.0 avobe, so official ReVanced Patches can be applied for YTM on Android 7.0.
Use original (or inotia00) source.

**Reply by MBYo** [reply]
Update: 
I already succeeded build the rooted version of the latest YTM and YT v.17.34.36 using inotia00's rvx-builder on rooted Android 7.0, For YT I was using your patches obviously, and YTM inotia00. 
Thanks arigatou gozaimasu for making the patches @kitadai31

**Reply by kitadai31** [reply]
No problem!
